opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/OpenTracingShim.java

```diff
@@ -11,6 +11,9 @@
 import io.opentelemetry.api.trace.TracerProvider;
 
 /**
+ * This class is deprecated. Please Use {@link OpenTracingShimBuilder} instead to create an
```

Comment:
Hmm. I wonder if it would be good to keep this class, just with the `builder()` method on it, to keep the patterns the same as they are in the rest of the code. It does make this class a bit anemic, but it was already that way. :)